### ISSUE TITLE
Should we add a default license?

### ISSUE THREAD

**Issue by frm** [subvisual/guides#42]
Following #41 I was wondering if it would be a fair idea to add a default LICENSE template for open source projects.

If so, any LICENSE suggestions?

**Comment by pfac** [comment]
I agree we should provide some guidance, but a default license would depend heavily. Client projects are rarely public, and thus rarely (if not never) need a license. Anything based on a GPL-licensed project will have to be GPL-licensed as well. The Ruby ecosystem, and I believe the Elixir ecosystem as well, default to MIT license. If you go after other ecosystems/projects you often see the Apache and Mozilla licenses.

My suggestion would be simply to provide instructions directing the authors to https://choosealicense.com. If we want to go to the trouble we might provide some help deciding between each license for Subvisual projects.

**Comment by frm** [comment]
Yeah, that seems a sensible option!

How would you feel about adding some steps paraphrasing what you just mentioned here, on the README template? We would also need to add some instructions to the GitHub section README as well.

**Comment by pfac** [comment]
Sure, I'll take a swing.
